porcupine/default_keybindings.tcl

```diff
@@ -3,11 +3,13 @@
 # Use Command on mac, Control on other systems
 if {[tk windowingsystem] == "aqua"} {
     set contmand Command
+    set anchmand Control
     event add "<<RightClick>>" <Button-2>
     event add "<<RightClick>>" <Control-Button-1>
     event add "<<WheelClick>>" <Button-3>
 } else {
     set contmand Control
+    set anchmand $anchmand
```

Comment:
- I don't like how these are named. Maybe `control_ish` and `alt_ish`?
- `set anchmand $anchmand` is similar to `anchmand = anchmand` in Python: it errors because you never defined what `anchmand` is. Maybe you meant `set anchmand Alt`, or `set alt_ish Alt` or whatever that will be?

Reply:
Yeah, I accidentally used replaced all when changing `Alt` to `$anchmand`. I can change it to alt_ish if you like it